### ISSUE TITLE
Proposal: support to run against secured etcd

### ISSUE THREAD

**Issue by supereagle** [contiv/netplugin#777]
In production, ETCD should be secured with TLS to ensure the security and stability, but Contiv currently does not support secured etcd. This is an essential feature when use Contiv in produtin, any plan to support this?

**Comment by jojimt** [comment]
Yes, this is one of the things we would like to do, but delayed due to resource crunch at the moment. Would be happy to accept a contribution on this.

**Comment by supereagle** [comment]
I am glad to contribute this work.

**Comment by rhim** [comment]
@supereagle any update?

**Comment by supereagle** [comment]
@rhim Sorry, I am not working on Contiv, and do not have enough time for this work.
